language/transaction-builder/generator/src/lib.rs

```diff
@@ -33,6 +33,7 @@ pub fn read_abis<P: AsRef<std::path::Path>>(dir_path: P) -> anyhow::Result<Vec<S
         abis.push(lcs::from_bytes(&buffer)?);
     }
     // Sorting scripts by alphabetical order.
+    #[allow(clippy::unnecessary_sort_by)]
```

Comment:
The borrow checker fails with sort_by_key as is suggested by this lint.   Probably indicates there are more issues with the borrowing occurring in this method.